scripts/utils/update_package.py

```diff
@@ -39,6 +43,15 @@ def get_latest_version(org, project, version):
     if not response.ok:
         print(f"GitHub API response not ok: {response.status_code}")
         return None
+    if org == "ufrisk" and project == "MemProcFS":
```

Comment:
It is not desirable to add code for only two packages, the code should be generic.